kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/jsonschema/AvroJsonSchemaConverter.java

```diff
@@ -55,12 +56,12 @@ private FieldSchema convertSchema(String name, Schema schema,
           if (schema.getType().equals(Schema.Type.MAP)) {
             return new MapFieldSchema(convertSchema(name, schema.getValueType(), definitions, ref));
           } else {
-            return createObjectSchema(name, schema, definitions, ref);
+            return createObjectSchema(schema, definitions, ref);
           }
         case ARRAY:
           return createArraySchema(name, schema, definitions);
         default:
-          throw new RuntimeException("Unknown type");
+          throw new KafkaUiRuntimeException("Unknown json type");
```

Comment:
that looks like fixing a code smell just for the sake of getting rid of a warning. Please invent some better named exception isntead :)

Reply:
Ok)